tests/conftest.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from jina import Document, DocumentArray
+
+from executor import VideoFrameExtractor
+
+
+@pytest.fixture()
+def encoder(tmp_path) -> VideoFrameExtractor:
+    workspace = str(tmp_path / 'workspace')
+    encoder = VideoFrameExtractor(metas={'workspace': workspace})
+    return encoder
+
+
+@pytest.fixture(scope='package')
+def build_da():
```

Comment:
Since this is only used once, just move it to the function where it is actually used